docs/concepts/privacy.md

```diff
@@ -0,0 +1,63 @@
+
+### Node IDs and node requests
+
+The other half of the equation when considering the prospect of IPFS traffic monitoring is that nodes' unique identifiers are themselves public. Just like with CIDs, every individual IPFS node has its own public identifier, such as `QmRGgYP1P5bjgapLaShMVhGMSwGN9SfYG3CM2TfhpJ3igE`.
+
+While a long string of letters and numbers may not be a "Johnny Appleseed" level of human-readable specificity, it's still a long-lived, unique identifier for your node. Keep in mind that it's possible to do a DHT lookup on your node's identifier and, particularly if your node is regularly running from the same location (like your home), find your IP address. Additionally, longer-term monitoring of the public IPFS network could yield information about what CIDs your node is requesting and/or reproviding and when.
+
+## Enhancing your IPFS privacy
+
+If there are situations in which you know you'll need to remain private but still want to use IPFS, one of the approaches outlined below may help. And don't forget, you can always discuss privacy and get others' input or ideas in the official [IPFS forums](https://discuss.ipfs.io).
+
+### Using a public gateway
+
+Using a public [IPFS gateway](https://dweb-primer.ipfs.io/avenues-for-access/review-these-lessons-from-the-tutorial-on-interacting-with-the-classical-http-web/public-gateways) is one way to request IPFS-hosted content without revealing any information about your local node — because you aren't using a local node! However, this method does keep you from enjoying all the benefits of being a full participant in the IPFS network.
+
+Public IPFS gateways are primarily intended as a "bridge" between the legacy web and the distributed web; they allow ordinary web clients to request IPFS-hosted content via HTTP. That's great for back-compatibility, but if you only request content through public gateways rather than directly over IPFS, you're not actually part of the IPFS network; that gateway is the network participant, acting on your behalf. It's also important to remember that gateway operators could be collecting their own private metrics, which could include tracking the IP addresses that use a gateway and correlating those with what CIDs are requested.
+
+### Using Tor
+
+If you're familiar with [Tor](https://www.torproject.org/), using the Tor browser and an IPFS-Tor gateway is a way to retrieve content from IPFS directly over Tor protocol. You can learn more about this experimental method in the [IPFS Primer](https://dweb-primer.ipfs.io/avenues-for-access/tor-gateways). If you're more adventurous and comfortable with the command line, you may also wish to try [running IPFS over Tor transport](https://dweb-primer.ipfs.io/avenues-for-access/tor-transport) by configuring your node's settings.
```

Comment:
Flag: dweb primer article is bit outdated, which creates issues:

* `ipfs4uvgthshqonk.onion` I don't believe we are running public gateweay on tor
  * https://ipfs4uvgthshqonk.onion/ipns/en.wikipedia-on-ipfs.org does not load

Reply:
Drat - removed material about the gateway for safety's sake in https://github.com/ipfs/ipfs-docs/pull/554/commits/21e0e929e925833acc85d50f848ef7bc41878df7.